WordPress/Classes/ViewRelated/Post/Revisions/RevisionsTableViewController.swift

```diff
@@ -34,9 +36,15 @@ class RevisionsTableViewController: UITableViewController {
         }
     }
 
-    convenience init(post: AbstractPost) {
-        self.init()
+
+    required init(post: AbstractPost, _ selectedRevisionLoaded: @escaping SelectedRevisionBlock) {
```

Comment:
I think `_` should have a name so that it appears in autocomplete suggestions (trailing closure syntax will still work)